labgrid/driver/rawnetworkinterfacedriver.py

```diff
@@ -97,6 +97,28 @@ def wait_state(self, expected_state, timeout=60):
         """Wait until the expected state is reached or the timeout expires."""
         self._wait_state(expected_state, timeout=timeout)
 
+    @Driver.check_active
+    def get_settings(self):
+        """
+        Returns settings via ethtool of the bound network interface resource.
+        """
+        cmd = self.iface.command_prefix + ["ethtool", "--json", self.iface.ifname]
+        output = subprocess.check_output(cmd, encoding="utf-8")
+        return json.loads(output)[0]
+
+    @Driver.check_active
+    @step(args=["settings"])
+    def configure(self, **settings):
```

Comment:
Perhaps these should be named after ethtool? Interfaces have many settings that are not configured via 'ethtool --change'.

Reply:
I've now renamed the methods.